### PR TITLE
added delay to get mail

### PR DESCRIPTION
**Pull request by birbalsain**
#### Describe Bug or Feature
In test "Test_array_job_email.test_email_non_existent_user"  recipient is not getting the mail message upon stageout failed because e-mail sent is taking time.

#### Describe Your Change
Added a loop with sleep 5 sec to get an e-mail delivered

#### Attach Test and Valgrind Logs/Output
before changes
Description: Tests from given sources on platforms SUSE15, CENTOS8, CENTOS7
Platforms: SUSE15,CENTOS8,CENTOS7
Profile: regression
|ID|TOTAL|FAIL|ERROR|TIMEDOUT|SKIP|PASS|
|--|--|--|--|--|--|--|
|6716|2|1|0|0|0|1|

Description: Tests from given sources on platforms SUSE15, CENTOS8, CENTOS7
Platforms: SUSE15,CENTOS8,CENTOS7
Profile: regression
|ID|TOTAL|FAIL|ERROR|TIMEDOUT|SKIP|PASS|
|--|--|--|--|--|--|--|
|6744|2|0|0|0|0|2|

### CODE REVIEW

File: test/tests/functional/pbs_array_job_mail.py [hunk 1]
```diff
@@ -76,18 +76,19 @@ def test_emails(self):
...
+                time.sleep(5)
+                ret = self.du.cat(filename=mailfile, sudo=True)
+                maillog = [x.strip() for x in ret['out'][-600:]]
+                for i in range(0, len(maillog) - 2):
+                    if jobid == maillog[i] and msg == maillog[i + 2]:
+                        emailpass = 1
+                        break
+                else:
+                    continue
+                break
```
Comment (bhroam) [inline test/tests/functional/pbs_array_job_mail.py:91]: You wouldn't need
```
else:
    continue
break
```
if you did:
```
if emailpass:
    break
```
at the end.

File: test/tests/functional/pbs_array_job_mail.py [hunk 2]
```diff
@@ -76,18 +76,19 @@ def test_emails(self):
...
         for (jobid, msg) in emails:
             emailpass = 0
-            for i in range(0, len(maillog)-2):
-                if jobid == maillog[i] and msg == maillog[i+2]:
-                    emailpass = 1
-
+            for j in range(5):
+                time.sleep(5)
+                ret = self.du.cat(filename=mailfile, sudo=True)
+                maillog = [x.strip() for x in ret['out'][-600:]]
```
Comment (bhroam) [inline test/tests/functional/pbs_array_job_mail.py:84]: If you only want the last 600 lines, maybe tail is a better option (like you use below).

**Review by bhroam — CHANGES_REQUESTED** [review]

**Comment by birbalsain** [comment]
Description: Tests from given sources on platforms SUSE15, CENTOS8, CENTOS7
Platforms: SUSE15,CENTOS8,CENTOS7
Profile: regression
|ID|TOTAL|FAIL|ERROR|TIMEDOUT|SKIP|PASS|
|--|--|--|--|--|--|--|
|6756|3|0|0|0|0|3|

**Review by bhroam — APPROVED** [review]
LGTM

**Review by kjakkali — APPROVED** [review]

**Pull request merged**